src/App/App.module.css

```diff
@@ -1,32 +1,108 @@
+body {
+  background: url(../img/bg.jpg);
+  padding: 0;
+  margin: 0;
+  animation: animate .3s linear infinite;
+}
+
 .wrap {
-  margin: auto;
-  width: 302px;
+  margin: 0 auto;
+  width: 340px;
   font-family: 'Montserrat', sans-serif;
+  background-color: #f7f7f7;
+  padding: 10px;
+  border-radius: 10px;
 }
 
 .title {
   color: #3f51b5;
+  position: relative;
 }
 
-.list {
-  font-style: italic;
+.add_task {
+  display: flex;
+  justify-content: space-between;
 }
 
-.footer {
-  font-weight: bold;
+.add_task button {
+  align-self: flex-end;
 }
 
-.done {
-  opacity: .8;
-  font-size: 80%;
-  text-decoration: line-through;
+.add_task > div {
+  margin-right: 20px;
 }
 
-.add_task {
-  display: flex;
-  justify-content: space-between;
+.wrap > * {
+  animation: shake .5s linear infinite;
 }
 
-.add_task > div {
-  margin-right: 20px;
+input {
+  animation-iteration-count: 1 !important;
 }
+
+body:hover {
+  background: #fff;
+}
+
+.wrap:hover *{
+  animation-iteration-count: 1;
+}
+
+@keyframes animate {
+  0% {
+    background-position: 0 0;
+  }
+  100% {
+    background-position: 100% 100%;
+  }
+}
+
+@keyframes shake {
+  0% {
+    -webkit-transform: translate(1px, 1px) rotate(0deg);
+   }
+  10% { 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/dotodo/pullRequest?prid=5840443)